packages/react-app/src/contexts/page.context.tsx

```diff
@@ -13,7 +13,7 @@ type Props = {
   children: React.ReactNode;
 };
 export const PageContextProvider = ({ children }: Props) => {
-  const [page, setPage] = useState(DEFAULT_PAGE);
+  const [page, setPage] = useState<string | undefined>(DEFAULT_PAGE);
```

Comment:
In fact we should instead use `DEFAULT_PAGE` just put undefined and remove that constants or change back to ENUM_PAGES.LIST as default.